database/index.js

```diff
@@ -1,3 +1,4 @@
 const config = require('../knexfile');
-const environment = process.env.NODE_ENV || 'development'
+const environment = process.env.NODE_ENV || 'development' //remove
 module.exports = require('knex')(config[environment]);
+// module.exports = require('knex')(config);
```

Comment:
In general you don't want to leave comments in when doing PRs, if anything it's more preferred to leave a `TODO` statement than commented out code:

`TODO: Refactor knex config to not use environment`

IDE's often have search for `TODO` keyword capabilities